simple_history/models.py

```diff
@@ -178,29 +178,18 @@ def copy_fields(self, model):
                 field.__class__ = models.IntegerField
             if isinstance(field, models.ForeignKey):
                 old_field = field
-                field_arguments = {'db_constraint': False}
+                old_field.swappable = False
```

Comment:
Let's change this back to its original setting after calling deconstruct

Reply:
@tbeadle let's change the swappable setting back after doing this deconstruct? And I'd love to figure out why those fields need to be sorted in tests – haven't been able to find why the order changes but want to know before merging. I'll look into it some more